### ISSUE TITLE
JAWS doesn't announce the label of a menu button (Chrome, Edge)

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#471]
### Summary

1. https://codepen.io/jaws-test/pen/yLJmWga
2. Tab to the buttons

### Expected result

label is output

### Actual result

* Edge/Chrome with JAWS: title is output instead of the label
* Correct output with NVDA or with Firefox+JAWS
* According to the Accessibility Tab of Chrome, Label is transmitted correctly. The error is probably related to the problem that button with aria-haspopup has the wrong role according to the specification (see https://github.com/w3c/core-aam/issues/51 and https://github.com/FreedomScientific/VFO-standards-support/issues/358)

#### JAWS version and build number
JAWS 2020.2008.24

 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 87.0.4280.66
